mods/core/src/common/resource_builder.ts

```diff
@@ -10,6 +10,7 @@ enum Kind {
 
 enum Privacy {
   PRIVATE = "Private",
+  PUBLIC = "Public",
```

Comment:
SIP only supports to Privacy status `id` and `private`. The correlation in Fonoster/Routr is set to be:

Private => `private`
None => `id`

Therefore, we need no Public, just Private and None.

Please see how this is used in Fonoster/Routr: https://github.com/fonoster/routr/blob/41659716f4951bbbb1f3fb848ee0dd6171ef0692/mod/core/processor/request_utils.js#L277